nextjs-app/app/offline/components/offline-projects.tsx

```diff
@@ -90,21 +104,27 @@ function OfflineProject({ i, project, openIndex, setOpenIndex }: {
 
 function ProjectRow({images, text}: ProjectRowProps) {
   return (
-    <div>
-      { images?.length && (
-        <ul>
+    <div className="space-y-8">
+      {images?.length && (
+        <ul className="space-y-4">
           {images.map((image) => {
             return (
-              <li key={image._key} >
-                <Image image={image} />
+              <li key={image._key} className="relative w-full h-[50vh]">
```

Comment:
we probably want images to show at their intrinsic aspect-ration as much as possible. in the case of two-images side by side this will be harder when they have different aspect ratios, but can be accomplished with flexbox and object-cover